### ISSUE TITLE
Missing 2nd & 3rd Phase Power SPH 6000 TL3 BH-UP

### ISSUE THREAD

**Issue by pimisen** [muppet3000/homeassistant-grott#55]
Hi & thanks a lot for Grott Integration!
I am missig the Values for Inverter Output Power (Second Phase) and Inverter Output Power (Third Phase)

**Comment by pimisen** [comment]
Seems to be an issue on grott-Server side, i gues as the values are not present in the MQTT Message received:
`{
    "device": "TLJ4CGA00R",
    "time": "2023-10-16T08:58:31",
    "buffered": "no",
    "values": {
        "pvstatus": 5,
        "pvpowerin": 7700,
        "pv1voltage": 2071,
        "pv1current": 21,
        "pv1watt": 4513,
        "pv2voltage": 4814,
        "pv2current": 6,
        "pv2watt": 3258,
        "pvpowerout": 898,
        "pvfrequentie": 4997,
        "pvgridvoltage": 3917,
        "pvgridcurrent": 6,
        **"pvgridpower": 4302,**
        "pvgridvoltage2": 4123,
        "pvgridcurrent2": 6,
        **"pvgridpower2": 0,**
        "pvgridvoltage3": 4035,
        "pvgridcurrent3": 6,
        **"pvgridpower3": 0,**
        "totworktime": 21156656,
        "eactoday": 21,
        "pvenergytoday": 21,
        "eactotal": 7897,
        "epvtotal": 10930,
        "epv1today": 1,
        "epv1total": 4570,
        "epv2today": 0,
        "epv2total": 6360,
        "pvtemperature": 359,
        "pvipmtemperature": 359,
        "pvboosttemp": 361,
        "bat_dsp": 1586,
        "eacharge_today": 14,
        "eacharge_total": 891,
        "batterytype": 1,
        "uwsysworkmode": 5,
        "systemfaultword0": 0,
        "systemfaultword1": 0,
        "systemfaultword2": 0,
        "systemfaultword3": 0,
        "systemfaultword4": 0,
        "systemfaultword5": 0,
        "systemfaultword6": 0,
        "systemfaultword7": 2048,
        "pdischarge1": 0,
        "p1charge1": 5700,
        "vbat": 1577,
        "SOC": 12,
        "pactouserr": 0,
        "pactousertot": 0,
        "pactogridr": 0,
        "pactogridtot": 0,
        "plocaloadr": 2100,
        "plocaloadtot": 2100,
        "spdspstatus": 5,
        "spbusvolt": 7233,
        "etouser_tod": 39,
        "etouser_tot": 458,
        "etogrid_tod": 0,
        "etogrid_tot": 2424,
        "edischarge1_tod": 9,
        "edischarge1_tot": 5058,
        "eharge1_tod": 1,
        "eharge1_tot": 5290,
        "elocalload_tod": 41,
        "elocalload_tot": 9258
    }
}

QoS: 0 - Retain: false `

**Comment by muppet3000** [comment]
Indeed, if you can get them mapped through into grott (probably worth opening a ticket over here: https://github.com/johanmeijer/grott/issues) then I can get them added in this integration if you need them

**Comment by pimisen** [comment]
Ticket on Grott Side: https://github.com/johanmeijer/grott/issues/458
